static/docs/command-reference/run.md

```diff
@@ -52,6 +52,14 @@ captures data and <abbr>caches</abbr> relevant <abbr>data artifacts</abbr> along
 the way. See [this example](/doc/get-started/example-pipeline) to learn more and
 try creating a pipeline.
 
+### Parallel execution
+
+When running your command, DVC will remove the project lock (`.dvc/lock` file),
+so that you will be able to run other DVC commands in parallel. However, it uses
+per-path read-write locking instead, to guarantee that no two DVC instances
+would be writing to the same path and don't write to paths that are being read
+from by another instance.
```

Comment:
> Continuation from https://github.com/iterative/dvc.org/pull/868#discussion_r358582191

We should probably mention (somewhere in the description above) that `.dvc/lock` is created while the command is running? Or maybe I don't understand how locking works now? Is there an explanation somewhere else? 🙂

Also, I'm still not sure we need an H3 header for this paragraph. It could just be the last paragraph in description. No strong opinion.

Reply:
I guess if we keep the H3 header then we can link to here directly from other docs, where appropriate. BTW, that's what I was talking about in 

> we can link existing instances of "always locked" phrase...

(back when this section was titled "Locking".)

Reply:
> Is there an explanation somewhere else?

I guess https://dvc.org/doc/command-reference/lock explains what locking is (so I would add a link to that doc from the word "lock" this paragraph BTW) but not really when it happens. Should that doc be expanded on and updated along with these changes? Or perhaps make a new short user guide about it, including this stuff about parallelism?

All of the above can also be extracted to a new issue if we want to get this to merge sooner. (Works for me.)

Reply:
@jorgeorpinel that link explains `dvc lock` command and not repo locks. Those are completely different things.

Reply:
There you go. So if I was confused, a new/avg user will probably not get the different locking mechanisms at all :/

So anyway, I'm extracting this discussion to new issue #880 but still please note my previous 2 comments. Thanks